node/src/effect.rs

```diff
@@ -974,6 +974,26 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    pub(crate) async fn get_activation_point_era_id(self) -> Option<u64>
```

Comment:
This is getting the block height, not era ID.

Reply:
Renamed to something less confusing... `get_key_block_height_for_activation_point`